src/main/java/com/softserve/sprintone/jose/util/Fraction.java

```diff
@@ -0,0 +1,17 @@
+package com.softserve.sprintone.jose.util;
+
+import java.math.BigInteger;
+
+public class Fraction {
+
+    private String fraction;
+
+    public Fraction(int numerator, int denominator) {
+
```

Comment:
despite it was a good idea to have this as a util class, I don't see this one is doing anything, if not used, please remove